playbooks/pki.yml

```diff
@@ -157,7 +158,7 @@
             state: directory
             mode: "{{ item.mode | default(omit) }}"
           loop:
-            - name: /etc/pki
+            - name: /etc/pki/{{ inventory_hostname | dirname }}
             - name: /etc/pki/private
               mode: "0600"
```

Comment:
this mode looks inappropriate. Do you mind fixing it while you're doing that?

Reply:
I changed it to 700. Let me know if you prefer 750 instead. Depending on distribution, it's either 700 or 750 for /etc/pki or /etc/ssl.